docs/examples/evaluation/prometheus_evaluation.ipynb

```diff
@@ -0,0 +1,1498 @@
+{
```

Comment:
<p>out of curiosity, have you experienced rate limit errors with this <code>batch_eval_runner</code>  when you using gpt-4? </p><p> </p><p>I realize now that we have <code>batch_eval_runner</code> and my <code>RagEvaluatorPack</code>  that are essentially doing similar things. However <code>RagEvaluatorPack</code> is defaulted to certain metrics and allows for controlling batches and sleep time to reduce RateLimitError  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9406/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
Nope. I did not face any rate limit errors with batch eval runner.

Reply:
Oh yeah. I was in tier-4. That should be the reason.